search-app/src/cmr/search/routes.clj

```diff
@@ -70,10 +71,17 @@
     mt/json
     mt/xml))
 
+(defconfig test-environment? 
+   "Flag that indicates if the environment is test environment"
+   {:default false :type Boolean})
+
 (def robots-txt-response
   "Returns the robots.txt response."
-  {:status 200
-   :body (slurp (io/resource "public/robots.txt"))})
+  (if (test-environment?) 
```

Comment:
Since this is done in a `def` it will look at the configuration at build time rather than deploy time.

We'll need to have two different vars and perform the `(test-environment)` check when the route is called.

Note that this also will let you add a test to verify the correct contents are returned when changing the configuration.

Reply:
Thanks. Added the test.